app/src/main/res/values/dimens.xml

```diff
@@ -140,4 +140,20 @@
     <dimen name="peril_icon_size">24dp</dimen>
     <dimen name="peril_card_size">56dp</dimen>
     <dimen name="logged_in_card_elevation">1dp</dimen>
+    <dimen name="peril_bottom_sheet_icon_size">48dp</dimen>
+    <dimen name="peril_bottom_sheet_close_icon_size">24dp</dimen>
+    <dimen name="peril_bottom_sheet_exception_icon_size">24dp</dimen>
+    <dimen name="peril_bottom_sheet_exception_item_size">56dp</dimen>
+    <!--
```

Comment:
wat this? 🤔

Reply:
Haha good question! I used the wearable activity Wizard by misstake and i guess it added this